scss/_popover.scss

```
@@ -203,15 +209,19 @@
   &:empty {
     display: none;
   }
-
-  // Boosted mod: popover body special padding when following a popover header
-  + .popover-body {
-    padding-top: var(--#{$prefix}popover-header-padding-bottom);
-  }
-  // End mod
 }
 
 .popover-body {
-  // Boosted mod: no `padding`
+  padding:
+    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-top))
+    var(--#{$prefix}popover-body-padding-x)
+    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-bottom))
+    var(--#{$prefix}popover-body-padding-x); // Boosted mod: instead of padding: `var(--#{$prefix}popover-body-padding-y) var(--#{$prefix}popover-body-padding-x);`
   color: var(--#{$prefix}popover-body-color);
 }
+
+// Boosted mod
+:not(.popover-header) + .popover-body {
```

I think the selector could be more specific to select popover-body here

```suggestion
.popover-arrow + .popover-body {
```